### PR TITLE
Fix quoting a few places in docs

### PR DESCRIPTION
**Pull request by tkelman**
Rerun doc/genstdlib.jl

and more line numbers in doctests [av skip]

I'm a small way into reviewing #12435, these are simple pre-existing formatting issues that need fixing. This will conflict with that PR but also make its output more correct if it gets rebased on top of this.

### CODE REVIEW

File: base/docs/helpdb.jl [hunk 1]
```diff
@@ -8503,7 +8503,7 @@ doc"""
 ::
            Base.compilecache(module::Symbol)
 
-Creates a precompiled cache file for module (see help for ``require``) and all of its dependencies. This can be used to reduce package load times. Cache files are stored in LOAD_CACHE_PATH[1], which defaults to `~/.julia/lib/VERSION`. See the manual section `Module initialization and precompilation` (under `Modules`) for important notes.
+Creates a precompiled cache file for module (see help for ``require``) and all of its dependencies. This can be used to reduce package load times. Cache files are stored in ``LOAD_CACHE_PATH[1]``, which defaults to ``~/.julia/lib/VERSION``. See the manual section ``Module initialization and precompilation`` (under ``Modules``) for important notes.
```
Comment (tkelman) [inline base/docs/helpdb.jl]: > ``` ``Module initialization and precompilation`` (under ``Modules``)```

should be using actual RST cross-references which I need to look up how to do

**Comment by tkelman** [comment]
cc @viralbshah bunch of examples here of pre-existing formatting issues in the docs, found while reviewing #12435, that are easier to find and fix before merging that than they would be after. Anywhere there are currently single-backquotes where there should be double would get lost in translation to markdown.

**Comment by ViralBShah** [comment]
So, is the idea to look at #12435 and have separate PRs to fix things on top of that - and then apply them all one after the other in quick succession?

**Comment by tkelman** [comment]
Well #12435 needs a rebase right this second, and if it gets rebased on top of this then these particular quoting problems will be fixed. Some or all of these I would have just pushed to master if it didn't introduce new conflicts there, but always better to have another set of eyes to check things.

**Comment by tkelman** [comment]
Will merge in a few hours if no feedback here.

**Pull request merged**